client/a8c-for-agencies/sections/signup/signup-v2/components/multi-step-form/contact-form/style.scss

```diff
@@ -0,0 +1,139 @@
+.signup-multi-step-form__fields {
+	display: flex;
+	flex-direction: column;
+	gap: 8px;
+	overflow-y: auto;
+	flex: 1;
+
+	// Customize scrollbar
+	&::-webkit-scrollbar {
+		width: 8px;
+	}
+
+	&::-webkit-scrollbar-track {
+		background: var(--studio-gray-0);
```

Comment:
It is better to avoid variables from the studio everywhere on this file, and instead, we should use `--color-neutral`. This will ensure we are using the color based on the theme.